### PR TITLE
Remove default option in npm install command

### PR DESCRIPTION
**Pull request by donhuvy**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

Remove default option in npm install command

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/7950)

### CODE REVIEW

**Comment by kenotron** [comment]
Hello @donhuvy, thanks for the PR. However, all up I would encourage that we keep the more verbose `install` syntax here because this is a doc for new comers. I'll link to two popular React component libraries and their tutorial sections:

https://material-ui.com/getting-started/installation/
https://ant.design/docs/react/introduce

**Comment by KevinTCoughlin** [comment]
> At this point, we can probably get rid of --save now that `npm` v5 is in the LTS. However, I would like to see this not include a change to `npm` shortcuts for `install`

Agreed. The verbose syntax highlighting changes are also not necessary unless we believe they help with readability.

File: README.md [hunk 1]
```diff
@@ -40,14 +40,14 @@ Integrating components into your project depends heavily on your setup. The reco
...
+npm i office-ui-fabric-react
 ```
 
-This will add the fabric-react project as a dependency in your package.json file, and will drop the project under node_modules/office-ui-fabric-react.
+This will add the fabric-react project as a dependency in your `package.json` file, and will drop the project under `node_modules/office-ui-fabric-react`.
 
 The library includes commonjs entry points under the lib folder. To use a control, you should be able to import it and use it in your render method:
 
-```js
+```javascript
```
Comment (ecraig12345) [inline README.md]: Curious of the reasoning for this and the ts/typescript change later? Markdown parsers ought to understand either as far as I'm aware.

Comment (donhuvy) [reply README.md]: Thank @ecraig12345 about code highlight `js`. I read here https://guides.github.com/features/mastering-markdown/ about code hightlight, I just see `javascript`, then I am not sure about `js`, and it is also follow a popular or easy-to-understand way.

**Comment by donhuvy** [comment]
> > At this point, we can probably get rid of --save now that `npm` v5 is in the LTS. However, I would like to see this not include a change to `npm` shortcuts for `install`
> 
> Agreed. The verbose syntax highlighting changes are also not necessary unless we believe they help with readability.

Thank @kenotron, about `install` and `--save`. Thank @KevinTCoughlin for `install`. In popular practices, also as my own practice, I follow instruction at top-right corner of page https://www.npmjs.com/package/office-ui-fabric-react is `npm i office-ui-fabric-react`.

**Comment by donhuvy** [comment]
All of these above things are not important and trivial changes. I will close this pull request. Thank you because I know more about the development of `office-ui-fabric-react`.

**Pull request closed without merging**